src/pages/login/loginStyle.tsx

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundImage: `linear-gradient(to top, #30cfd0 0%, #330867 100%)`,
+    "& .MuiContainer-root": {
```

Comment:
not good way
https://material-ui.com/api/container/
just pass a class for the container root